src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
+
+![Screenshot of a memory actions]({% asset tools/devtools/memory_actions.png @path %}){:width="100%"}
+
+<dl markdown="1">
 <dt markdown="1">**Snapshot**</dt>
-<dd markdown="1">Display a table of current active
-    memory objects (see
-    [Snapshot classes](#snapshot-classes)).
-<dt markdown="1">**Reset accumulator**</dt>
-<dd markdown="1">Reset values, to zero, under the
-    Accumulator column in the Snapshot classes table.
-<dt markdown="1">**Filtering classes**</dt>
-<dd>PENDING</dd>
-<dt markdown="1">**Snapshot classes**</dt>
-<dd>Clicking the Snapshot button (top right area)
-    displays a table of current memory objects.
-    The memory objects can be sorted by class name,
-    size, allocated instances, and so on.</dd>
-<dt markdown="1">**Accumulator counts since reset**</dt>
-<dd>Clicking the Reset button (top right area) resets
-    the accumulated instances count. Clicking Snapshot
-    after a reset displays the number of new instances
-    allocated since last reset.
-    This is useful for finding memory leaks.</dd>
-<dt markdown="1">**Class instances**</dt>
-<dd>Clicking a class in the Snapshot class table
-    displays the number of active instances for that class.</dd>
-<dt markdown="1">**Inspecting contents of an instance**</dt>
-<dd>PENDING</dd>
-<dt markdown="1">**Total active objects and classes in the heap**</dt>
-<dd>Total classes allocated in the heap and total objects
-    (instances) in the heap.</dd>
-
-## Memory overview chart
+<dd markdown="1">Display a table of all current active
+    memory objects.  Clicking the Snapshot button makes a request to the Dart VM to collect the current state of memory.  A snapshot displays a table of current memory objects. The memory objects can be sorted by class name, size, allocated instances, and so on.(see [Snapshot classes](#snapshot-classes)).
+<dt markdown="1">**Treemap**</dt>
+<dd markdown="1">Display all collected snapshots, analysis and monitor in a Table Tree view (most-detailed) default (see [Snapshot classes](#snapshot-classes)).  If Treemap is switched on then the last snapshot, high-level data, is disabled as a tree map. (TBD details).</dd>
+<dt markdown="1">**Group By**</dt>
+<dd markdown="1">Dropdown to select how trees are ordered for display by instance or by class name.</dd>
+<dt markdown="1">**Collapse All**</dt>
+<dd markdown="1">Collapse all nodes in the Table Tree view.</dd>
+<dt markdown="1">**Expand All**</dt>
+<dd markdown="1">Expand all nodes in the Table Tree view.</dd>
+<dt markdown="1">**Monitor Allocations**</dt>
+<dd markdown="1">User clicked on Monitor Allocations.  To record the number of instances and size of all instances in bytes.  The accumulators (Delta column) can be used to track memory allocated since last reset.</dd>
```

Comment:
"Records the number of instances and size of all instances in bytes. Upon clicking on the "Monitor" entry in the table, another table will populate with instance allocation data. For each instance in the allocation table, The "Delta" column reflects the number of memory allocations since the last reset."

Reply:
Done.